Calla/package.json

```diff
@@ -3,13 +3,17 @@
     "version": "0.11.0",
     "homepage": "https://calla.chat",
     "scripts": {
-        "build": "rollup -c --environment BUILD:production",
+        "build": "run-script-os",
+        "build:darwin:linux": "cp README.md .. && rollup -c --environment BUILD:production",
+        "build:win32": "copy README.md .. && rollup -c --environment BUILD:production",
         "build:patch": "npm version patch && node writeversion.js && npm run build",
         "build:minor": "npm version minor && node writeversion.js && npm run build",
         "build:major": "npm version major && node writeversion.js && npm run build",
         "watch": "rollup -c -w --environment BUILD:development"
     },
-    "dependencies": {},
+    "dependencies": {
+        "run-script-os": "^1.1.3"
+    },
```

Comment:
This should be a devDependency, not a dependency. No worries, I'll just switch it over.